src/components/Shared/FormElements/AddressFormComponents/ManualEntry.test.tsx

```diff
@@ -65,8 +65,13 @@ describe("ManualEntry", () => {
         target: { value: "test town" },
       });
 
+      fireEvent.input(screen.getByLabelText("Postcode"), {
+        target: { value: "test postcode" },
```

Comment:
now that a real postcode is required, is this causing a failure? use a real postcode format 
e.g. PA London, SW1E 5DN